### ISSUE TITLE
Accessibility: problematic screen reader UX for ContextMenuStrip (NVDA)

### ISSUE THREAD

**Issue by jscholes** [dotnet/winforms#3334]
* .NET Core Version: 3.1
* Have you experienced this same bug with .NET Framework?: No

#### Problem description:

When using the standard `ContextMenu` removed in https://github.com/dotnet/winforms/pull/2157, NVDA announces "Context Menu" when the menu opens.  With `ContextMenuStrip`, NVDA doesn't announce anything on pop-up (not even the presence of a menu without an accessible name), meaning that a user has no idea that a menu has been invoked.

#### Expected behavior:

NVDA will announce "Context Menu" on pop-up.

#### Minimal repro:

```
using System.Drawing;
using System.Windows.Forms;

namespace Test {
	public class MainWindow : Form {
		public MainWindow() {
			this.ClientSize = new Size(800, 450);
			this.Text = "Context Menu Accessibility";
			this.AutoScaleDimensions = new SizeF(6F, 13F);
			this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Font;
			this.SuspendLayout();
			this.InitializeControls();
			this.ResumeLayout(false);
			this.PerformLayout();
		}

		private void InitializeControls() {
			Button aButton = new Button{
				Text="Button with a ContextMenuStrip"
			};
			ContextMenuStrip ctxMenu = new ContextMenuStrip();
			ctxMenu.Items.Add("Item &1");
			ctxMenu.Items.Add("Item &2");
			ctxMenu.Items.Add("Item &3");
			aButton.ContextMenuStrip = ctxMenu;
			this.Controls.Add(aButton);
		}

		protected override void Dispose(bool disposing) {
			base.Dispose(disposing);
		}
	}
}
```

**Comment by jscholes** [comment]
Note: it's likely that this problem also affects `MenuStrip` when using a menu-bar-style setup, but I didn't include it in the main issue description because I haven't tested it.

**Comment by jscholes** [comment]
Confirmed that JAWS announces "context menu" as expected, but NVDA does not.

**Comment by RussKie** [comment]
@vladimir-krestov @Ryuugamine can you have a look?

**Comment by SergeySmirnov-Akvelon** [comment]
Hi @merriemcgaw. I can reproduce this issue in the NVDA:
![Issue-3334-NVDA](https://user-images.githubusercontent.com/23376742/99064736-6a718700-25b7-11eb-9fa1-27394bf0f129.gif)

But the Narrator works correctly for same case:
![Issue-3334-Narrator](https://user-images.githubusercontent.com/23376742/99064718-60e81f00-25b7-11eb-9d7e-97a5ecf1f15a.gif)

Probably, it is external issue of the NVDA. Should we try to fix it for the NVDA?

**Comment by SergeySmirnov-Akvelon** [comment]
Hi @merriemcgaw. After opening the context menu, it and its options are available in the inspector and Accessibility Insights
![Issue-3334-contextmenu](https://user-images.githubusercontent.com/23376742/100237876-90e0dc00-2f40-11eb-937a-d2e5e871a6b1.gif)

**Comment by merriemcgaw** [comment]
Can you follow up with Guy in the Teams site? It would be best to have a snapshot of the UIA tree either from Accessibility insights or this. I think we should be allowed to close this.

**Comment by merriemcgaw** [comment]
@jscholes This appears to be a problem with how NVDA is interpreting our ContextMenu information. We've validated that we're doing everything on our end that we can. Please file an issue with NVDA about this problem.

**Comment by jscholes** [comment]
@merriemcgaw 

> Please file an issue with NVDA about this problem.

Will do!  Thanks for looking into it.

**Comment by merriemcgaw** [comment]
@jscholes Any time 😄
